decision-records/005-repository-organisation-for-v5.md

```diff
@@ -0,0 +1,120 @@
+# Decision Record Title
+
+Update our repository organisation to prepare v5
+<!-- The title should be a short present tense imperative phrase, less than 50
+     characters, like a git commit message. -->
+
+## Decision
+
+<!-- An overview of the decision made. -->
+
+To prepare our work for v5, we've agreed to make the following changes to the files in our repository and their location:
+
+1. Move the sources (currently `src` ) and built package files (currently `package`) under the same npm package. We'll keep shipping only the built files to npm for now, excluding the source files from the package (either via `.npmignore` or using the `files` field of `package.json`)
+2. Remove the built package files (currently `package`) from the versioning
+3. Update our release process to make sure we keep checking the changes (possibly manually) between the package to be released and the previous version
+4. Leave `dist` in its current location
```

Comment:
Moving `dist` isn't a priority for us currently and we'd unnecessarily lose a handy diff workflow

But I'm not sure it's fair to say we've decided to leave it

Reply:
I'll add a "for now" to nuance as we may want to come back to that in the future (wouldn't mind removing it from version control at some point, for ex). I thought worthy to mention it to make sure the scope of the changes was clearly limited to "package" when it came to built files.